guacamole-docker/bin/start.sh

```diff
@@ -830,6 +870,12 @@ if [ -n "$OPENID_AUTHORIZATION_ENDPOINT" ]; then
     INSTALLED_AUTH="$INSTALLED_AUTH openid"
 fi
 
+# Use SAML if specified
+if [ -n "$SAML_IDP_METADATA_URL" ]; then
```

Comment:
This check will need to be more complicated, since the `saml-idp-metadata-url` property can be omitted entirely so long as the required information is given with other properties.

Overall, the user needs either `saml-idp-metadata-url`, or `saml-entity-id` and `saml-callback-url`, or a combination of these if the metadata XML is incomplete.

Reply:
> [...]if the metadata XML is incomplete

That will be hard to check from within a bash script as I would need to parse the content of saml-idp-metadata-url to check for what is missing and if it is present in the other variables.

Can't really say when I will get to this. If someone else wants to make the needed changes and needs permissions on my fork, simply let me know.

Reply:
I wouldn't recommend making things that complex or attempting to duplicate the logic that the extension itself will be using.

I think it would be far better to just relax the logic a bit and consider the presence of any of those environment variables sufficient to enable SAML, rather than actually parse the XML.

Reply:
I've implemented the logic.